fim/user/topology.py

```diff
@@ -750,6 +754,14 @@ def __list_links(self) -> ViewOnlyDict:
     def sites(self):
         return self.__list_sites()
 
+    @property
+    def nodes(self):
+        """
+        Same as sites property
+        :return:
+        """
+        return self.__list_sites()
```

Comment:
Question: Is this intended to return sites?

Reply:
Yes in advertised topologies it is the same as .nodes property.